conf/eslint.json

```diff
@@ -56,6 +56,7 @@
         "no-multiple-empty-lines": [0, {"max": 2}],
         "no-native-reassign": 0,
         "no-negated-in-lhs": 2,
+        "no-negated-instanceof-lhs": 2,
```

Comment:
We cannot add new recommended rules without a major version upgrade. Please set to 0.